app/views/ErrorInternalView.scala.html

```diff
@@ -0,0 +1,28 @@
+@*
```

Comment:
Can we call this InternalServerErrorTemplate instead, as it seems to be specifically used for 500 internal server errors